### PR TITLE
More turbine groups (1,2,3,4,6,8,12)

### PR DESCRIPTION
**Pull request by ufechner7**
- [x] increase T_SKIP to 1500s
- [x] add optimized parameters for 2, 3, 8, and 12 turbine groups
- [x] add code to create videos, skipping T_SKIP
- [x] add test for NaNs in the measurements; they happen if the induction is zero
- [x] add function create_n_groups to the package
- [x] add documentation to mpc.jl in the form of comments
- [x] remove one constraint
- [x] add vis::Vis as parameter to a number of calc_induction related functions

### CODE REVIEW

File: examples/plot_turbines.jl [hunk 1]
```diff
@@ -85,6 +86,63 @@ function plot_turbines(ta::TurbineArray, turbine_groups)
...
+    
+    # Add "all" group
+    push!(turbine_groups, Dict(
+        "name" => "all",
+        "id" => 0,
+        "turbines" => collect(1:n_turbines)
+    ))
+    
+    return turbine_groups
+end
```
[the remark this answers is not in the document]
Comment (ufechner7) [reply examples/plot_turbines.jl]: I moved create_n_groups to the package and I am reusing it here.

File: docs/src/map_inductions.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+Let the relative power demand be defined by the function:
+$$
+d = g(t) \\
+0 < d < 1.0
+$$
+
+We can write:
+$$
+a_i = f(i, t) \\
+
```
[the remark this answers is not in the document]
Comment (ufechner7) [reply docs/src/map_inductions.md]: No need for it to render correctly in all MarkDown processors.

File: examples/calc_induction_matrix.jl [hunk 3]
```diff
@@ -166,7 +165,7 @@ Includes group-based corrections and time interpolation.
 # Returns
 - Axial induction factor for the specified turbine
 """
-function calc_axial_induction(ta, con, turbine, time; correction_factor=1.8, dt=DT) # max 1.8
+function calc_axial_induction(ta, con, turbine, time; correction_factor=1.8) # max 1.8
```
[the remark this answers is not in the document]
Comment (ufechner7) [reply examples/calc_induction_matrix.jl]: Fixed

File: src/settings.jl [hunk 4]
```diff
@@ -1559,6 +1560,90 @@ function select_measurement()
...
+# Returns
+- `TurbineArray`: A new turbine array with the same positions, types, and initial states,
+  but with updated groups divided by X coordinate
+
+# Algorithm
+1. Sort all turbines by their X coordinate (column 1 of `ta.pos`)
+2. Divide sorted turbines into `n_groups` groups of approximately equal size
+3. Distribute any remainder turbines to the first groups
+4. Create a new turbine array with these groups plus an "all" group containing all turbines
+
```
[the remark this answers is not in the document]
Comment (ufechner7) [reply src/settings.jl]: Fixed.
